okhttp/build.gradle

```diff
@@ -29,8 +29,16 @@ dependencies {
   compileOnly deps.jsr305
   compileOnly deps.animalSniffer
 
+  testImplementation deps.okio
+  testImplementation project(':okhttp-testing-support')
+  testImplementation project(':okhttp-tls')
+  testImplementation project(':okhttp-urlconnection')
+  testImplementation project(':mockwebserver')
+  testImplementation project(':okhttp-logging-interceptor')
```

Comment:
And this one...

Reply:
EventListenerTest.java uses it. Can fix, but not this PR.